addons/point_of_sale/models/pos_config.py

```diff
@@ -355,6 +355,23 @@ def _check_payment_method_receivable_accounts(self):
                 _("You must configure an intermediary account for the payment methods: %s.") % method_names
             )
 
+    @api.constrains('module_pos_loyalty', 'loyalty_id')
+    def _check_loyalties_product_availability(self):
+        if self.module_pos_loyalty and self.loyalty_id:
+            archived_reward_products = {}
+            for loyalty in self.loyalty_id:
+                for reward in loyalty.reward_ids.filtered(
+                        lambda r: r.reward_type == "resale" and r.point_product_id and not r.point_product_id.active):
+                    archived_reward_products.setdefault(loyalty, []).append(reward.point_product_id)
+            if archived_reward_products:
+                t = ""
+                for loyalty, products in archived_reward_products.items():
+                    t += "\nLoyalty \"%s\" (id %s):" % (loyalty.name, loyalty.id)
```

Comment:
`t += _("\nLoyalty \"%s\" (id %s):") % (loyalty.name, loyalty.id)`
use _() for translation

Reply:
done with the updated version. Thank you for the proposition